cli/command_policy_set_splitter.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"context"
+	"sort"
+
+	"github.com/alecthomas/kingpin/v2"
+
+	"github.com/kopia/kopia/repo/splitter"
+	"github.com/kopia/kopia/snapshot/policy"
+)
+
+type policySplitterFlags struct {
+	policySetSplitterAlgorithmOverride string
+}
+
+func (c *policySplitterFlags) setup(cmd *kingpin.CmdClause) {
+	cmd.Flag("splitter", "Splitter algorithm override").EnumVar(&c.policySetSplitterAlgorithmOverride, supportedSplitterAlgorithms()...)
+}
+
+//nolint:unparam
+func (c *policySplitterFlags) setSplitterPolicyFromFlags(ctx context.Context, p *policy.SplitterPolicy, changeCount *int) error {
+	if v := c.policySetSplitterAlgorithmOverride; v != "" {
+		if v == inheritPolicyString {
```

Comment:
Q: IIUC, `inheritPolicyString` is used in order for resetting a policy setting when it was already set in a child, is that the case? Otherwise, what's the difference between "inheriting" and an unset value?